packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -244,14 +266,65 @@ export class WaferMap extends FoundationElement {
                 this.dataManager.updateDiesRenderInfo();
                 this.renderer.updateSortedDiesAndDrawWafer();
             } else if (this.waferMapUpdateTracker.requiresDrawnWaferUpdate) {
-                this.renderer.drawWafer();
+                await this.renderer.drawWafer();
             }
+            await this.drawWafer();
             this.zoomHandler.connect();
```

Comment:
indeed, I left it there since the beginning to be sure it will be always called, I moved it to a better place 
![image](https://github.com/ni/nimble/assets/110180309/2058180f-b5d4-4af4-b751-8ed23a9694a5)

Reply:
Just to make sure I'm not missing something, are you intending to move `this.zoomHandler.connect()`, or were you referring to something else?